recipes/motmetrics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "motmetrics" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7328d8468c948400b38fcc212f3e448bc1f2fdfc727e170d85a029e49f1cdbc6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - flake8
```

Comment:
Sorry, I just noticed this. Why are you requiring `flake8`, `flake8-import-order`, `pytest`, and `pytest-benchmark`? These are developer tools that based on [motmterics requirements.txt](https://github.com/cheind/py-motmetrics/blob/develop/requirements.txt) are not actually required to run the package.

Reply:
this was generated with grayskull (which I assume pulled them in from the motmetrics requirements_dev.txt file) -- not sure what the standards for conda-forge is for dev dependencies but more than happy to remove

Reply:
We're ok in general for including optional dependencies that give the library it's full capabilities. In this case, they only seem necessary to adhere to style and run unit tests if you're developing on the library. I say go ahead and remove them.

I hadn't used grayskull, so had no idea.

Reply:
@dopplershift should I remove them completely, or just put them as test dependencies (putting them as test dependencies seems better to me, but I have used/worked with conda-forge for publishing before so take it with a grain of salt)

Reply:
I'd remove them completely. Test dependencies are dependencies that are needed to execute the test steps in the recipe, which in this case is just importing the package. Tests in a recipe are really just there to try to prevent broken packages, so I don't think there's any reason to run the full test suite--unless you disagree.

Reply:
fine by me!

Reply:
@dopplershift I think those packages might have been necessary because the test just failed: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=198760&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=819

I'm going to add them back as test dependencies unless there's a way to bypass this that you know of

Reply:
I had no idea pip check would do that. Yeah, I guess tests deps is the way to go.

Reply:
Sorry, I should have been clear: Add them under `requires` in the `test` section lower down.

Reply:
no worries! my fault for no reading the docs carefully enough